### ISSUE TITLE
Kyverno Policy reports are not shown in UI

### ISSUE THREAD

**Issue by m-yosefpor** [kyverno/policy-reporter-kyverno-plugin#7]
I have deployed the kyverno policy reporter with official helm charts with the following values:

```  
  ui:
    enabled: true
    plugins:
      kyverno: true
    ingress:
      enabled: true
      hosts:
      - host: myhost.example.com
        paths:
        - path: /
  kyvernoPlugin:
    enabled: true
```

And I can see the policies in UI:
<img width="1147" alt="image" src="https://user-images.githubusercontent.com/47300215/137650062-c8e84a49-4671-466e-8305-0a479c2bc3fb.png">

Also I can see the policy warnings in the `Logs` section:

<img width="1419" alt="image" src="https://user-images.githubusercontent.com/47300215/137650093-f2ca3088-8354-417a-ae0a-f8382c3e2313.png">

However, there are no reports in other part of the UI:
<img width="736" alt="image" src="https://user-images.githubusercontent.com/47300215/137650124-52c64d14-3d8c-49d6-8c8f-f931afab9710.png">

<img width="1167" alt="image" src="https://user-images.githubusercontent.com/47300215/137650143-93df4190-988a-453a-950c-5465439c75d8.png">

**Comment by fjogeleit** [comment]
Hey, thanks for your Report. Which version of kyverno do you running?

can you please check the logs of the Policy Reporter Pod?

**Comment by fjogeleit** [comment]
Hey, thank you. I released a new version of the Policy Reporter Image. Can you try it out in your installation by setting the image version to 1.10.0.

```yaml
image:
  tag: 1.10.0
```

If this doesn't help, can you check if the API calls are successful in the UI and if they contain data?

<img width="1784" alt="Bildschirmfoto 2021-10-18 um 11 56 32" src="https://user-images.githubusercontent.com/16627596/137710077-9f46c8b6-d14a-4c93-938f-af3ea2a5be79.png">

**Comment by m-yosefpor** [comment]
@fjogeleit thank you very much for the help. I think I've found the issue. Upgrading to 1.12.2 didn't help. The api calls in network call are all `500`  http error.

<img width="851" alt="image" src="https://user-images.githubusercontent.com/47300215/137792999-03c3098e-5500-420f-a71f-edf5a0b4240a.png">


With the following response:
```
{ "message": "Get "http://policy-reporter:8080/cluster-policy-reports": dial tcp: lookup policy-reporter on 172.30.0.10:53: no such host" }
```

The issue is I have deployed the helm chart with another release name `cloud-policy-reporter` instead of `policy-reporter`, so all svc name are prefixed with  `cloud-` and the lookup is failing. I didn't know it has a dependency to chart release name (and I think it shouldn't have such dependency), and right now it seems it must be only equal to `policy-reporter`.
